src/categories/components/CategoryUpdatePage/CategoryUpdatePage.tsx

```diff
@@ -96,8 +100,23 @@ export const CategoryUpdatePage: React.FC<CategoryUpdatePageProps> = ({
       {({ data, change, handlers, submit, isSaveDisabled }) => (
         <DetailPageLayout gridTemplateColumns={1}>
           <TopNav href={backHref} title={category?.name}>
+            <Button
+              variant="secondary"
+              icon={<TranslationsIcon />}
+              onClick={() =>
+                navigate(
+                  languageEntityUrl(
+                    locale.toLocaleUpperCase(),
+                    TranslatableEntities.categories,
+                    categoryId,
+                  ),
+                )
+              }
+            />
             {extensionMenuItems.length > 0 && (
-              <TopNav.Menu items={[...extensionMenuItems]} dataTestId="menu" />
+              <Box marginLeft={3}>
+                <TopNav.Menu items={[...extensionMenuItems]} dataTestId="menu" />
```

Comment:
question: Why do we need to create new array on each render? 🤔 

```ts
[...extensionMenuItems]
```

Reply:
I don't know, it's not part of my changes ;)